app/controllers/schools/low_carbon_hub_installations_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Schools
+  class LowCarbonHubInstallationsController < ApplicationController
+    load_and_authorize_resource :school
+    load_and_authorize_resource through: :school
+
+    def index
+      @low_carbon_hub_installations = @school.low_carbon_hub_installations
```

Comment:
Doesn't the above `through` do this already?

Reply:
yes, affirmative